CMakeLists.txt

```diff
@@ -12,20 +12,17 @@ HunterGate(
 
 # project configuration
 set(PROJECT_NAME_STR pandora)
-project(${PROJECT_NAME_STR} VERSION "0.10.0" LANGUAGES C CXX)
+project(${PROJECT_NAME_STR} VERSION "0.11.0" LANGUAGES C CXX)
```

Comment:
We're okay to stay on 0.10.0 as we are still in alpha release, which basically says we are in the process of adding features for 0.10.0. Best not to skip 0.10.0 as we never did a stable release yet.

Reply:
Done in https://github.com/rmcolq/pandora/pull/318/commits/a6f7db0ef50a3926d3c3e8ae971cb10941b103c7